modin/pandas/test/utils.py

```diff
@@ -671,7 +671,11 @@ def execute_callable(fn, inplace=False, md_kwargs={}, pd_kwargs={}):
                 # repr to force materialization
                 repr(fn(modin_df, **md_kwargs))
             if check_exception_type:
-                assert isinstance(md_e.value, type(pd_e))
+                assert isinstance(
+                    md_e.value, type(pd_e)
+                ), "Got Modin Exception type {}, but pandas Exception type {}".format(
```

Comment:
This isn't super clear - perhaps say `Got Modin Exception type {}, but pandas Exception type {} was expected`?